guides/v2.1/config-guide/cli/config-cli-subcommands-i18n.md

```diff
@@ -15,30 +15,36 @@ Magento translations enable you to customize and localize your store for multipl
 
 This topic discusses how to generate:
 
--   Translation dictionaries, which are a convenient way to translate *some* words and phrases, such as those for a custom {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %} or {% glossarytooltip d2093e4a-2b71-48a3-99b7-b32af7158019 %}theme{% endglossarytooltip %}.
+-   Translation dictionaries, which are a convenient way to customize or translate *some* words and phrases, such as those for a custom {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %} or {% glossarytooltip d2093e4a-2b71-48a3-99b7-b32af7158019 %}theme{% endglossarytooltip %}.
 -   Language packages, which enable you to translate *any or all* words and phrases in the Magento application.
 
-For more information, see [Translation]({{ page.baseurl }}/frontend-dev-guide/translations/xlate.html).
+For more information, see [Translations overview].
+
+We also accept [Community Engineering contributions] using CrowdIn for translations. We plan to provide translation packs through the project. Join us and other contributors around the globe working to localize Magento.
```

Comment:
Also, we accept...

When you say "plan to provide", it commits you to a future endeavor. Is this okay?

Reply:
Yeah we do have plans rolling out in Q1 2019, the how and what not is being finalized. I may rephrase just in case.